test/utils.py

```diff
@@ -76,10 +78,13 @@ def setup_experiment(installed=True):
             pypath = ':'.join([sdk_path, cmd_path])
         os.environ['PYTHONPATH'] = pypath
 
-def fetch_nni_log_path(experiment_url):
+def get_experiment_id(experiment_url):
+    experiment_id = requests.get(experiment_url).json()['id']
+    return experiment_id
+
+def get_nni_log_path(experiment_url):
```

Comment:
why refactor this place?

Reply:
For convenience. Since I need a new function to get experiment id to be used in other place
and the first two lines of this function is to get experiment id, so...
BTW, I change `fetch` to `get` to be more consistent with other functions' name